src/ONE.Models.CSharp/ONE.Models.CSharp.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
```

Comment:
Hmmm I might be wrong here but I think the protobuf changes need to be linked here

Reply:
I can hop on a call with you to fix this